AUTHORS.txt

```diff
@@ -0,0 +1,2 @@
+Dylan Vaughn <dylancvaughn@gmail.com>
+Adam Burns <adam@operatingops.org>
```

Comment:
@operatingops should we use our OE emails?

Reply:
@dylanvaughn I usually try to match the authors lines with what shows up in git commit messages, but using our OE emails might make more sense here. Either way is good, I think. Want me to swap them over?

Reply:
@operatingops yes let's use oe addresses